electron/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "unified-auctions",
+  "version": "1.0.0",
+  "description": "MakerDAO Unified Auctions UI",
+  "author": "SIDESTREAM GmbH",
+  "main": "main.js",
+  "build": {
+    "appId": "com.MakerDAO.UnifiedAuctions",
+    "mac": {
+      "category": "public.app-category.finance"
+    }
+  },
+  "scripts": {
+    "preinstall": "cd ../frontend/ && npm ci && npm run build && npm run generate && cd ../electron/",
```

Comment:
Name `preinstall` is not just a name, it's a special script name that will be run every time after npm installation. Is it desired?

> These all run after the actual installation of modules into node_modules, in order, with no internal actions happening in between

https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-operation-order